src/test/java/edu/hm/hafner/util/StringContainsUtilsTest.java

```diff
@@ -30,5 +30,13 @@ void shouldSearchStrings() {
         assertThat(containsAnyIgnoreCase("This is a string text.", "this")).isTrue();
         assertThat(containsAnyIgnoreCase("This is a string text.", "wrong", "is")).isTrue();
         assertThat(containsAnyIgnoreCase("This is a string text.", "wrong", "IS")).isTrue();
+        assertThat(containsAllIgnoreCase("This is a string text.", "something")).isFalse();
```

Comment:
Hier wären auch Tests sinnvoll die das Handling von null-Referenzen testen. Zum Beispiel so wie in StringContainsUtilsTest#shouldHandleNull().